packages/common/src/Environments.ts

```diff
@@ -72,7 +74,8 @@ const ganacheLocal: Environment = {
   mintxgascost: 21000,
   gtxdatanonzero: 16,
   gtxdatazero: 4,
-  getGasPriceFactor: 1
+  getGasPriceFactor: 1,
+  nonZeroDevOverhead: 7847
```

Comment:
1. I hate the idea of more gas estimation params. Should be based on estimateGas. 
2. But even if we do add it, we need a way to calculate it.

Reply:
The way to calculate it is not different than the current way we do it, through tests, logging the diff, then fixing the gasOverhead based on it.